src/main/java/org/openRealmOfStars/player/combat/CombatAnimation.java

```diff
@@ -365,7 +321,23 @@ public void doAnimation() {
       }
     } 
   }
-  
+
+  /* @TODO: Rename this parameter. */
```

Comment:
Parameter is used to determine animation frame when to place particle effects on hit animation.